build.ps1

```diff
@@ -14,6 +14,10 @@ param(
     [switch]$SkipLinkCheck
 )
 
+if ($isWindows) {
+    winrm quickconfig -quiet
```

Comment:
We should do this in the test and not the build script.  Probably in a `BeforeAll {}` block

Reply:
fixed